<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>parametrepage</class>
 <widget class="QDialog" name="parametrepage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(25, 25, 25)</string>
  </property>
  <widget class="QLabel" name="labTitre">
   <property name="geometry">
    <rect>
     <x>285</x>
     <y>30</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Paramètre réseaux</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbutIpAuto">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>331</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Obtenir une addresse IP automatiquement</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbutIpMan">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>331</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Paramétrer manuellement l'adresse IP</string>
   </property>
  </widget>
  <widget class="QLabel" name="labAddIp">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>220</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Addresse Ip :</string>
   </property>
  </widget>
  <widget class="QLabel" name="labMasque">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>270</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Masque :</string>
   </property>
  </widget>
  <widget class="QLabel" name="labPass">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>304</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Passerelle par </string>
   </property>
  </widget>
  <widget class="QLabel" name="labPass2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>320</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>defaut :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ip">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>215</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(48, 48, 48);
color: white;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>15</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="masque">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>260</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(48, 48, 48);
color: white;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>15</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="passerelle">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>305</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(48, 48, 48);
color: white;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>15</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="butValide">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>360</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color : white;
background-color:rgb(48, 48, 48)</string>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butAnnule">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>360</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color : white;
background-color:rgb(48, 48, 48)</string>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
